src/routes/api/authRoute.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import userController from '../../controllers/UserController';
+import isUserExist from '../../middlewares/FindUsers';
+import validateUser from '../../middlewares/ValidateUser';
+
+const authRouter = express.Router();
+
```

Comment:
Please refactor the route to have just `/signup` and add auth to the route index